src/dto/common.dto.ts

```diff
@@ -3,3 +3,8 @@ export interface ResDto {
   msg?: string;
   error?: string;
 }
+
+export enum ProbeType {
+  SNOWFLAKE = 'snowflake',
+  BIGQUERY = 'bigquery',
```

Comment:
```suggestion
  SNOWFLAKE = 'SNOWFLAKE',
  BIGQUERY = 'BIGQUERY',
```
always good to keep values of enum in capital.